src/test/java/com/amihaiemil/zold/RtWalletTestCase.java

```diff
@@ -199,4 +199,38 @@ public void findsWalletTransactions() throws Exception {
             Matchers.equalTo(payload)
         );
     }
+
+    /**
+     * Checks whether the {@link RtWallet#pay(String, String, String, String)}
+     * invocation succeeds.
+     * @throws Exception If an exception is thrown
+     */
+    @Test
+    public void pays() throws Exception  {
+        final ClassicHttpResponse response = new Response(
```

Comment:
@ammaratef45 ,  could you please check the indentation for the new code? It should have been 4 spaces I think.